daemon/src/db/cull_dlcs.rs

```diff
@@ -0,0 +1,187 @@
+use crate::db::Connection;
+use anyhow::Result;
+use model::EventKind;
+
+impl Connection {
+    #[allow(dead_code)]
+    async fn cull_old_dlcs(&self) -> Result<()> {
```

Comment:
🙃 I see that you cull all DLCs but the last one. Can we change this to keeping the last two until rollover is more stable?

Reply:
Why are the last two important?

Reply:
I don't know why we would want to save anything other than the last one. What are you thinking about?

Reply:
Given that some logic around revoke commits is still missing it might be useful to have the old DLC so we can "cleanup"  CFDs by going one back in history. 🤷‍♂️

Reply:
Thanks for raising this!

It's true that we had that idea, but we haven't actually used it despite having a few out-of-sync CFDs. I don't think it's a critical feature (it's only a workaround for a bug) so I wouldn't adapt this new feature for it at this stage.

We can always reconsider before we release this new functionality.